pkg/kubelet/rkt/rkt.go

```diff
@@ -614,21 +618,49 @@ func (r *runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, er
 	glog.V(4).Infof("rkt: Creating service file %q for pod %q", serviceName, pod.Name)
 	serviceFile, err := os.Create(serviceFilePath(serviceName))
 	if err != nil {
-		return "", err
+		return "", nil, err
 	}
 	defer serviceFile.Close()
 
 	_, err = io.Copy(serviceFile, unit.Serialize(units))
 	if err != nil {
-		return "", err
+		return "", nil, err
 	}
 
 	if needReload {
 		if err := r.systemd.Reload(); err != nil {
-			return "", err
+			return "", nil, err
+		}
+	}
+	return serviceName, runtimePod, nil
+}
+
+// generateEvents is a helper function that generates some container
+// life cycle events for containers in a pod.
+func (r *runtime) generateEvents(runtimePod *kubecontainer.Pod, reason string, unit string, failure error) {
+	// Set up container references.
+	for _, c := range runtimePod.Containers {
+		id := string(c.ID)
+		ref, ok := r.containerRefManager.GetRef(id)
+		if !ok {
+			glog.Warningf("No ref for container %q", id)
+			continue
+		}
+
+		switch reason {
+		case "created":
+			r.recorder.Eventf(ref, "created", "Created with %q, unit file %q", id, unit)
```

Comment:
Reason should be CamelCase. Also please align the reasons you listed here with what we had done for docker container runtime:

https://github.com/kubernetes/kubernetes/blob/a945785409d5b68f3a2721d2209300edb5abf1ce/pkg/kubelet/dockertools/manager.go#L671

So Created reason means the same thing no matter which runtime is enabled. Same for rest of reasons in events.